components/authz-service/server/v2/rules_test.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 const applied = "applied"
+const staged = "staged"
```

Comment:
I'm aware your following the example of the code that was there already -- but it's a bit silly what we're doing here, isn't it? What do we try to protect against? I can only imagine two things:
1. We're renaming "applied" to "foobared" some time in the future. Would we then change this to `const applied = "foobared"`? I don't think so.
2. We're afraid of having typos in the code below, someone fat-fingering this into "aplied". What would happen? The tests where this is typo'ed in would probably fail, and the typo would be noticed. I'm having a hard time imagining a situation where a typo (which has had to slip through code review, too) turns a test into a phantom test, or some other harmful thing...

🤷‍♂ Of course it doesn't hurt either. I'm just wondering.

Reply:
I agree in general. I followed the pattern because I didn't wanna update all the places applied was already used without a convo because I assume it was an intentional decision. I also think it's very unlikely that string would change but I guess this is helpful if it does. Regardless, probably not worth updating at this point 😅